### ISSUE TITLE
Surface privacy implications of reporting issues / screenshots

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#2690]
Right now things are hidden in a privacy link. We could show some info with a link to that before users report a bug.

**Comment by miketaylr** [comment]
Something as simple as "All information included in this report will be publicly visible. Learn more" (which links to our privacy policy).

Amazing mockups here (low-res, but not a ton of detail is needed to get the picture): 
![screen shot 2019-02-06 at 5 02 22 pm](https://user-images.githubusercontent.com/67283/52379888-6b17d980-2a31-11e9-8998-c7df2e414998.png)

Thoughts or suggested improvements @karlcow?

**Comment by karlcow** [comment]
@miketaylr 

* no comment on the prose
* the yellow bar probably should be the width of the two buttons for the visual consistency. 
* I would do the same on desktop and chrome. 

![mockup](https://user-images.githubusercontent.com/505230/52384048-2c306500-2abf-11e9-88d6-db94a2821948.jpg)

**Comment by karlcow** [comment]
```html
<div class="form-text js-Form-group">
  <div class="privacy-warning">
    All information included in this report will be publicly visible.
    <a href="https://webcompat.com/privacy">Learn More</a>
  </div>
</div>
```

**Comment by miketaylr** [comment]
I agree that looks much nicer. Thank you @karlcow.

**Comment by karlcow** [comment]
Closed by #2795
